packages/react-components/src/components/ErrorBar.tsx

```diff
@@ -243,7 +258,11 @@ export const ErrorBar = (props: ErrorBarProps): JSX.Element => {
     [props.activeErrorMessages, dismissedErrors]
   );
 
-  const toShow = errorsToShow(props.activeErrorMessages, dismissedErrors);
+  const toShow = errorsToShow(
+    props.activeErrorMessages,
+    dismissedErrors,
+    props.ignorePremountErrors ? mountTimestamp.current : undefined
```

Comment:
Simply curious, is there a reason we decide to use a time stamp to ignore premount error instead of just clear all error in premounting when ignorePremountErrors === true?

Reply:
We can't clear any errors because they are stored in the stateful client. Uni-directional data flow implies that errors only clear when backend clears them (e.g. UFD backed errors)